tetris.js

```diff
@@ -100,7 +102,11 @@ module.exports = {
 					}
 					newCoordenats.push(obj)
 				}
-				return newCoordenats
+				if(checkThatTheFieldIsFree(newCoordenats)){
+					return localTetromino
+				}else{
```

Comment:
fix formatting.
VSCode will do this for you automatically

Reply:
formatting is correct, there is no error in my file

Reply:
No, there should be a space after '}' and the space '{' everywhere.
Please insteall VScode - it will fix this

Reply:
https://stackoverflow.com/a/29973358/362721